ocsp.go

```diff
@@ -567,30 +786,19 @@ func createOCSPCacheDir() {
 func init() {
 	readCACerts()
 	createOCSPCacheDir()
-	readOCSPCacheFile()
+	initOCSPCache()
 }
 
-// snowflakeInsecureTransport is the default tranport object that doesn't do certificate revocation check.
+// snowflakeInsecureTransport is the transport object that doesn't do certificate revocation check.
 var snowflakeInsecureTransport = &http.Transport{
 	MaxIdleConns:    10,
 	IdleConnTimeout: 30 * time.Minute,
 	Proxy:           http.ProxyFromEnvironment,
 }
 
-// SnowflakeTransport includes the certificate revocation check with OCSP in parallel. By default, the driver uses
+// SnowflakeTransport includes the certificate revocation check with OCSP in sequential. By default, the driver uses
 // this transport object.
 var SnowflakeTransport = &http.Transport{
-	TLSClientConfig: &tls.Config{
-		RootCAs:               certPool,
-		VerifyPeerCertificate: verifyPeerCertificateParallel,
```

Comment:
Parallel check was removed to simplify the code. No performance advantage was observed in my experiments.